### PR TITLE
Bug 1171459 – Prevent screenshots and history of about pages.

### PR DESCRIPTION
**Pull request by jhugman**

### CODE REVIEW

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 1]
```diff
@@ -1384,7 +1384,7 @@ extension BrowserViewController: WKNavigationDelegate {
 
         tab.expireSnackbars()
 
-        if let url = webView.URL where !ErrorPageHelper.isErrorPageURL(url) {
+        if let url = webView.URL where !ErrorPageHelper.isErrorPageURL(url) && url.scheme != "about" {
```
Comment (st3fan) [inline Client/Frontend/Browser/BrowserViewController.swift]: Is this correct? I thought we completely got rid of `about:` style URLs actually. Did they not get replaced with `http://localhost:*/` style URLs?

Comment (st3fan) [inline Client/Frontend/Browser/BrowserViewController.swift]: Yeah I don't see any references to `about:` in the current code anymore. For example `about:home` has now become `http://localhost:1234/about/home/#panel=0` instead.

Comment (fluffyemily) [inline Client/Frontend/Browser/BrowserViewController.swift]: There is now an `AboutUtils` struct with a `isAboutHome(url: NSURL)` method that could probably be used here

Comment (st3fan) [inline Client/Frontend/Browser/BrowserViewController.swift]: @jhugman See bove. Can you use that utility class?

**Comment by jhugman** [comment]
Use AboutUtils as suggested.

**Comment by fluffyemily** [comment]
oops, sorry. Pressed wrong button :worried:

**Comment by fluffyemily** [comment]
This looks good, but I don't like the 57 changed files to change 1 line.
Can you do:

```git checkout master
git pull --rebase
git checkout jhugman/Bug1171459-HideAboutPages-HomeTiles
git rebase master
git push -f origin jhugman/Bug1171459-HideAboutPages-HomeTiles```

And that should leave just your commit (hopefully)

**Pull request merged**